src/hooks/useAdvanceStep.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react'
+
+function useAdvanceStep() {
+  const [currentStep, setCurrentStep] = useState(0)
+
+  const scrollTop = () => window.scrollTo({ x: 0 })
+  const changeTitle = title => (document.title = title)
+  const setFocus = () => {
+    setTimeout(() => {
+      const focusTarget = document.getElementById('focus-target')
```

Comment:
Instead of working with `getElementById` I would prefer to use `querySelector` in combination with a `data` attribute. But that is up to you to decide. :)